test/protractor.conf.js

```diff
@@ -5,7 +5,7 @@ module.exports.config = {
   capabilities: {
     browserName: "chrome",
     chromeOptions: {
-      args: ["--headless", "--disable-gpu", "--window-size=1024,768"]
```

Comment:
In the Google Developers documentation they mention that `--disable-gpu` is temporarily needed if running on Windows, so I'll leave it here still.
For reference: https://developers.google.com/web/updates/2017/04/headless-chrome

Reply:
My computer is Windows and the headless work without the argument `--disable-gpu`. I'm sure it's not necessary.

Reply:
Still, let's follow what the documentation suggests.

Reply:
Done